### PR TITLE
Fix typos in README.md

### PR DESCRIPTION
**Pull request by anthonyfok**
Hello!  Thank you so much for creating goldmark!  It looks awesome!

I was in the process of packaging goldmark for Debian, and noticed a few typos in README.md while preparing for the package description, hence this pull request.

Thanks again!
Anthony

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,48 +6,48 @@ goldmark
...
     - Markdown has many dialects.
-    - Github Flavored Markdown is widely used and it is based on CommonMark aside from whether CommonMark is good specification or not.
+    - GitHub Flavored Markdown is widely used and it is based on CommonMark aside from whether CommonMark is good specification or not.
         - CommonMark is too complicated and hard to implement.
 - Well structured.
     - AST based, and preserves source position of nodes.
 - Written in pure Go.
 
-[golang-commonmark](https://gitlab.com/golang-commonmark/markdown) may be a good choice, but it seems copy of the [markdown-it](https://github.com/markdown-it) .
+[golang-commonmark](https://gitlab.com/golang-commonmark/markdown) may be a good choice, but it seems to be a copy of the [markdown-it](https://github.com/markdown-it).
```
Comment (aletson) [inline README.md]: Removing "the" seems better here

File: README.md [hunk 2]
```diff
@@ -6,48 +6,48 @@ goldmark
...
 
-[blackfriday.v2](https://github.com/russross/blackfriday/tree/v2) is a fast and widely used implementation, but it is not CommonMark compliant and can not be extended from outside of the package since it's AST is not interfaces but structs. 
+[blackfriday.v2](https://github.com/russross/blackfriday/tree/v2) is a fast and widely used implementation, but it is not CommonMark compliant and cannot be extended from outside of the package since its AST uses not interfaces but structs.
 
-Furthermore, its behavior differs with other implementations in some cases especially of lists.  ([Deep nested lists don't output correctly #329](https://github.com/russross/blackfriday/issues/329), [List block cannot have a second line #244](https://github.com/russross/blackfriday/issues/244), etc).
+Furthermore, its behavior differs from other implementations in some cases especially of lists.  ([Deep nested lists don't output correctly #329](https://github.com/russross/blackfriday/issues/329), [List block cannot have a second line #244](https://github.com/russross/blackfriday/issues/244), etc).
```
Comment (aletson) [inline README.md]: "in some cases, especially regarding lists", maybe? The comma is necessary, but I'm not sure about the rewording

File: README.md [hunk 3]
```diff
@@ -6,48 +6,48 @@ goldmark
...
 
-This behavior sometimes causes problems. If you migrate your markdown text to blackfriday based wikis from Github, many lists will immediately be broken.
+This behavior sometimes causes problems. If you migrate your Markdown text to blackfriday-based wikis from GitHub, many lists will immediately be broken.
 
 As mentioned above, CommonMark is too complicated and hard to implement, So Markdown parsers based on CommonMark barely exist.
```
Comment (aletson) [inline README.md]: uncapitalize So

File: README.md [hunk 4]
```diff
@@ -234,21 +234,21 @@ Summary:
 
 Security
 --------------------
-By default, goldmark does not render raw HTMLs and potentially dangerous urls.
+By default, goldmark does not render raw HTMLs and potentially dangerous URLs.
```
Comment (aletson) [inline README.md]: "raw HTML" - remove s

File: README.md [hunk 5]
```diff
@@ -234,21 +234,21 @@ Summary:
...
 
 Benchmark
 --------------------
 You can run this benchmark in the `_benchmark` directory.
 
 ### against other golang libraries
 
-blackfriday v2 seems fastest, but it is not CommonMark compiliant so performance of the
-blackfriday v2 can not simply be compared with other Commonmark compliant libraries.
+blackfriday v2 seems the fastest, but it is not CommonMark compliant, so the performance of
```
Comment (aletson) [inline README.md]: "seems to be" perhaps

File: README.md (hunk 3 above)
Comment (anthonyfok) [reply README.md]: Hi @aletson, thank you for all your great suggestions!
I'm sorry I didn't see your reviews until now, i.e. after this PR has been merged.
Let's make another pull request to incorporate your copyediting/proofreading!  :-)

Comment (aletson) [reply README.md]: Sure. I was out of town the last few days, I will get to this shortly.

**Pull request merged**